ext/levenshtein/levenshtein.c

```diff
@@ -104,3 +104,15 @@ int levenshtein_extern(char* a, char* b, int max_distance)
 
   return distance;
 }
+
+float normalized_levenshtein_extern(char* a, char* b, int max_distance)
+{
+  int a_len = (a == NULL) ? 0 : strlen(a);
+  int b_len = (b == NULL) ? 0 : strlen(b);
+
+  int max_string_length = (a_len > b_len) ? a_len : b_len;
+  if(max_string_length == 0) {
+    return 0;
```

Comment:
@abrandoned This return is if both strings are blank.

This line will return the length of the longest string to the method I created, which will result in dividing by itself and returning 1: https://github.com/mxenabled/vladlev/blob/ferg/normalize_and_bug_fix/ext/levenshtein/levenshtein.c#L75